src/core_io.h

```diff
@@ -26,7 +26,7 @@ std::vector<unsigned char> ParseHexUV(const UniValue& v, const std::string& strN
 
 // core_write.cpp
 std::string FormatScript(const CScript& script);
-std::string EncodeHexTx(const CTransaction& tx, const int serializeFlags = 0);
+std::string EncodeHexTx(const CTransaction& tx, const int serialFlags = 0);
```

Comment:
Here, I prefer the longer name. Flags are about the serialization not about the "serial" number or so. But kets wait for other opinions before changing it right away ;-)